client/platform/web-girder/views/ShareTab.vue

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { useStore } from '../store/types';
+
+
+export default defineComponent({
+  name: 'ShareTab',
+  props: {
+    value: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup() {
+    const store = useStore();
+    const locationStore = store.state.Location;
+    const { getters } = store;
+
+    const clearSelected = () => {
+      locationStore.selected = [];
+    };
+
+    return {
+      locationStore,
+      getters,
+      clearSelected,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-tabs
+    dense
+    right
+    class="px-4"
+    @change="clearSelected"
+  >
+    <v-tab :to="getters['Location/locationRoute']">
+      <v-icon class="mr-2">
+        mdi-folder-multiple
+      </v-icon>
+      Browse Data
+    </v-tab>
+    <v-tab :to="{name: 'shared'}">
+      <v-icon class="mr-2">
```

Comment:
There's a problem here where the width of the icon is dynamic, which causes a race condition with the calcuation of the underline active tab style.  Soemtimes it ends up in the wrong place because the icon hasn't rendered yet.

You can solve this by forcing a fixed with on the icon.  replace `class="mr-2"` with `class="tab-icon"` and define the new style as scoped.

```css
<style scoped>
.tab-icon {
  width: 28px;
  margin-right: 10px;
}
</style>
```

Reply:
Great catch.